### PR TITLE
aw - Resolve Dependencies Issues with Initial Pull

### PR DESCRIPTION
**Pull request by austinwong3**
Fixes CI issue for #10

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -20,9 +20,6 @@ Layout/MultilineMethodCallIndentation:
 Metrics/LineLength:
   Max: 112  # Maximum line length from `rails new`
 
-Rails:
-  Enabled: true
-
```
Comment (ivailov) [inline .rubocop.yml:25]: Why did you remove this setting?

**Comment by jschofel** [comment]
FYI there is an Adep version of ropes that allows you to deploy to QA
https://github.com/appfolio/newhire_app

File: .rubocop.yml (hunk 1 above)
Comment (austinwong3) [reply .rubocop.yml:25]: I misinterpreted the error that was being flagged. Since then I have followed your recommendations and added the rails cop back in.

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.6.3'
+ruby '2.5.3'
```
Comment (jschofel) [inline Gemfile:2]: Why did this get changed to a lower version?  I don't think we would ever want to downgrade the ruby version

Comment (austinwong3) [reply Gemfile:2]: 2.5.3 was the original version in the repo, I had upgraded it temporarily to iterate locally before I had `asdf` running and I forgot to change it back before one of my commits earlier. DDM mentioned that I should be matching my local to what was in the repo rather than changing the repo's version

File: .rubocop.yml [hunk 3]
```diff
@@ -8,10 +8,12 @@ AllCops:
     - 'bin/*'
   TargetRubyVersion: 2.5
 
-Layout/IndentFirstArrayElement:
```
Comment (jschofel) [inline .rubocop.yml:11]: Minor styling comment, but `require` statements are usually at the top of the file (but its not always the case).

Screenings Rubocop file: https://github.com/appfolio/screenings_app/blob/master/.rubocop.yml

Example of the inverse case: https://github.com/appfolio/screenings_app/blob/master/test/test_helper.rb

**Comment by grahamotte** [comment]
Couple thoughts: 

- The commits here follow different formats, some prefixed with `aw -` and others not
- Looks like some commits could be squashed, or just removed like `Adding rails back into rubocop` and `Removed Rails from Rubocop`

File: .rubocop.yml (hunk 3 above)
Comment (austinwong3) [reply .rubocop.yml:11]: Gotcha! I've moved the `require` to the top. Thanks!

**Review by grahamotte — APPROVED** [review]

**Review by jschofel — APPROVED** [review]
